tests/ui/test_ui_tools.py

```diff
@@ -3,12 +3,13 @@
 from typing import Any, Dict
 
 import pytest
+import pytz
 from bs4 import BeautifulSoup
 from urwid import Columns, Divider, Padding, Text
 
 from zulipterminal.config.keys import is_command_key, keys_for_command
 from zulipterminal.config.symbols import (
-    QUOTED_TEXT_MARKER, STREAM_TOPIC_SEPARATOR,
+    QUOTED_TEXT_MARKER, STREAM_TOPIC_SEPARATOR, TIME,
```

Comment:
I am not sure if we should include the date aspect because it only servers for time mentions. I have renamed it to `TIME_MENTION_MARKER` which perhaps should be more appropriate.

Yes, it [does](https://zulipchat.com/help/format-your-message-using-markdown#mention-a-time).
> ...you can mention a time, and it'll be displayed to each user in their own timezone (just like the timestamps on Zulip messages).